driver/src/main/scala/com/stratio/sparta/driver/SpartaJob.scala

```diff
@@ -380,28 +310,85 @@ object SpartaJob extends PolicyUtils {
     }).toMap
   }
 
-  def getSparkConfigFromPolicy(policy: PolicyModel) : Map[String, String] =
-    policy.sparkConf.flatMap{ sparkProperty =>
-      if(sparkProperty.sparkConfKey.isEmpty || sparkProperty.sparkConfValue.isEmpty)
-          None
-      else Option((sparkProperty.sparkConfKey, sparkProperty.sparkConfValue))}.toMap
-
-  def logAndCreateEx(message: String,
-                     json: String,
-                     ex: Throwable,
-                     policyId: String,
-                     code: Int): IllegalArgumentException = {
-    log.error(message)
-    log.error(s"JSON: $json")
-    ex.getCause match {
-      case cause: ClassNotFoundException =>
-        log.error("The component couldn't be found in classpath. Please check the type.")
-      case _ =>
-        log.error("Error instantiating policy component:", ex)
+  def getSparkConfigFromPolicy(policy: PolicyModel): Map[String, String] =
```

Comment:
This function should be moved to policyUtils, and StreamingContextService extends PolicyUtils